students/Sean_Tasaki/Lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,15 @@
+for i in range(1,101):
+	if (i % 3 == 0) and not (i % 5 == 0):
+		print("Fizz")
+	
+	elif (i % 5 == 0) and not (i % 3 == 0):
+		print("Buzz")
+	
+	elif (i % 5 == 0) and (i % 3 == 0):
```

Comment:
If you place this statement in the first `if` block, you can remove the `and not` clauses from the the other statements, since you will have already checked for numbers divisible by both 3 and 5